pkg/trtl/namespaces.go

```diff
@@ -3,8 +3,8 @@ package trtl
 // Reserved namespaces that cannot be used by the caller since they are in use by trtl.
 // If necessary this can be moved to configuration in the future.
 var reservedNamespaces = map[string]struct{}{
-	"peers":    struct{}{},
-	"sequence": struct{}{},
-	"index":    struct{}{},
-	"default":  struct{}{}, // if the user does not specify a namespace
+	"peers":    {},
+	"sequence": {},
+	"index":    {},
+	"default":  {}, // if the user does not specify a namespace
```

Comment:
My linter suggested this, so I went ahead and accepted the suggestion.

Reply:
Interesting that the GitHub lint didn't catch this since the linter in my VSCode complains about this as well.

Reply:
Perhaps because it's just a warning?

Reply:
my linter also complained, but did not make any suggestions -- @bbengfort what linter extension are you using for vscode?

Reply:
This I think: https://marketplace.visualstudio.com/items?itemName=golang.Go with the default tools. Could just be a version difference.